test/InventoryStoreSpec.js

```diff
@@ -1,129 +1,115 @@
+    it(`Should retrieve 2 of ${NUM_SINGLE_AVAILABILITY_LISTINGS} ${TEST_MARKET} listings available on ${SINGLE_AVAILABILITY_DATE}`, (done) => {
+      db.getAvailableListings(TEST_MARKET, SINGLE_AVAILABILITY_DATE, SINGLE_AVAILABILITY_DATE, 2)
         .then((results) => {
-          results.forEach((result) => {
-            availableListings.add(result.listing_id);
-          });
-          expect(availableListings.size).to.equal(2);
+          expect(results.length).to.equal(2);
           done();
         });
     });
 
-    it('Should retrieve all 4 San Francisco listings available on 2017-11-12 when top 100 is requested', (done) => {
-      db.getAvailableListings('San Francisco', '2017-11-12', '2017-11-12', 100)
+    it(`Should retrieve all ${NUM_SINGLE_AVAILABILITY_LISTINGS} ${TEST_MARKET} listings available on ${SINGLE_AVAILABILITY_DATE} when top 100 is requested`, (done) => {
+      db.getAvailableListings(TEST_MARKET, SINGLE_AVAILABILITY_DATE, SINGLE_AVAILABILITY_DATE, 100)
         .then((results) => {
           expect(results.length).to.equal(4);
           done();
         });
     });
 
-    it('Should retrieve 1 of 2 San Francisco listings available for all dates in the range 2017-11-10 to 2017-11-12', (done) => {
-      const stayDates = { '2017-11-10': 0, '2017-11-11': 0, '2017-11-12': 0 };
-      db.getAvailableListings('San Francisco', '2017-11-10', '2017-11-12', 1)
+    it(`Should retrieve 1 of ${NUM_RANGE_AVAILABLE_LISTINGS} ${TEST_MARKET} listings available for all dates in the range ${AVAILABLE_DATE_RANGE_START} to ${AVAILABLE_DATE_RANGE_END}`, (done) => {
+      db.getAvailableListings(TEST_MARKET, AVAILABLE_DATE_RANGE_START, AVAILABLE_DATE_RANGE_END, 1)
         .then((results) => {
-          results.forEach((result) => {
-            stayDates[result.inventory_date.toJSON().split('T')[0]] += 1;
-          });
-          Object.keys(stayDates).forEach((date) => {
-            expect(stayDates[date]).to.equal(1);
-          });
+          expect(results[0].dates.length).to.equal(AVAILABLE_DATE_RANGE_LENGTH);
+          expect(results.length).to.equal(1);
           done();
         });
     });
 
-    it('Should retrieve all 2 San Francisco listings available for all dates in the range 2017-11-10 to 2017-11-12 when top 100 is requested', (done) => {
-      const availableListings = new Set();
-      const stayDates = { '2017-11-10': 0, '2017-11-11': 0, '2017-11-12': 0 };
-      db.getAvailableListings('San Francisco', '2017-11-10', '2017-11-12', 100)
+    it(`Should retrieve all ${NUM_RANGE_AVAILABLE_LISTINGS} ${TEST_MARKET} listings available for all dates in the range ${AVAILABLE_DATE_RANGE_START} to ${AVAILABLE_DATE_RANGE_END} when top 100 is requested`, (done) => {
+      db.getAvailableListings(TEST_MARKET, AVAILABLE_DATE_RANGE_START, AVAILABLE_DATE_RANGE_END, 100)
         .then((results) => {
           results.forEach((result) => {
-            availableListings.add(result.listing_id);
```

Comment:
Shouldn't this be result => ( for Airbnb?

Reply:
No, there is no return value following it.